consensus/babe/runtime.go

```diff
@@ -39,3 +43,74 @@ func (b *Session) configurationFromRuntime() error {
 
 	return err
 }
+
+// calls runtime API function Core_initialize_block
+func (b *Session) initializeBlock(blockHeader []byte) error {
+	// TODO: use allocator to store block header
+	var loc int32 = 1
+	b.rt.Store(blockHeader, loc)
+
+	_, err := b.rt.Exec(runtime.CoreInitializeBlock, loc, blockHeader)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// calls runtime API function BlockBuilder_inherent_extrinsics
+//nolint:typecheck
+func (b *Session) inherentExtrinsics(blockInherentData []byte) ([]byte, error) { //nolint:unused
+	// TODO: use allocator to store inherents data
+	var loc int32 = 1
+	b.rt.Store(blockInherentData, loc)
+
+	return b.rt.Exec(runtime.BlockBuilderInherentExtrinsics, loc, blockInherentData)
+}
+
+// calls runtime API function BlockBuilder_apply_extrinsic
+//nolint:typecheck
+func (b *Session) applyExtrinsic(e types.Extrinsic) ([]byte, error) { //nolint:unused
+	// TODO: use allocator to store extrinsic
+	var loc int32 = 1
+	b.rt.Store(e, loc)
+
+	return b.rt.Exec(runtime.BlockBuilderApplyExtrinsic, loc, e)
+}
+
```

Comment:
Only `//nolint:unused` seems to be needed here

Reply:
ok removed typecheck